mixins/imageHelper.ts

```diff
@@ -64,14 +96,156 @@ declare module 'vue/types/vue' {
         maxWidth,
         maxHeight,
         quality,
-        limitByWidth
+        limitByWidth,
+        backdropIndex,
+        checkParent
       }: ImageUrlForElementParams
-    ) => string;
+    ) => string | undefined;
+    getParentId(item: BaseItemDto): string | undefined;
+    getImageTag(
+      item: BaseItemDto | BaseItemPerson,
+      type: ImageType,
+      index?: number,
+      checkParent?: boolean
+    ): string | undefined;
+    getBlurhashHash(
+      item: BaseItemDto | BaseItemPerson,
+      type: ImageType,
+      index?: number,
+      checkParent?: boolean
+    ): string | undefined;
+  }
+}
+
+/**
+ * Checks if the passed object is BaseItemDto or BaseItemPerson
+ *
+ * @param {BaseItemDto|BaseItemPerson} obj - The object to check
+ * @returns {boolean} Returns true if the object is a person, false otherwise.
+ */
+function isPerson(obj: BaseItemDto | BaseItemPerson): obj is BaseItemPerson {
+  if ((obj as BaseItemPerson)?.Role) {
+    return true;
   }
+  return false;
 }
 
+const excludedBlurhashTypes = [ImageType.Logo];
+
 const imageHelper = Vue.extend({
   methods: {
+    /**
+     * Gets the tag of the image of an specific item and type.
+     *
+     * @param {BaseItemDto|BaseItemPerson} item - The item object.
+     * @param {ImageType} type - The type of the image requested.
+     * @param {number} [index=0] - Index of the Backdrop image (when ImageType equals to 'Backdrop').
+     * @param {boolean} [checkParent=true] - Looks for tag/image type for the parent if the passed item doesn't have the ImageType requested
+     * @returns {string|undefined} Returns the tag, undefined if the specific ImageType doesn't exist.
+     */
+    getImageTag(
+      item: BaseItemDto | BaseItemPerson,
+      type: ImageType,
+      index = 0,
+      checkParent = true
+    ): string | undefined {
+      if (item) {
+        if (!isPerson(item)) {
+          if (item.ImageTags?.[type]) {
```

Comment:
What happens here when an item has a parent and should show its parent image, but also has images of its own?

Reply:
@mrtimscampi own item images will always take precedence, so parents will be ignored. We can add an option to force query for parents if we need it.

Reply:
That was my concern, there are a few situations where we'd want the parent's images instead of the items :)

Reply:
I said adding another option for that when I was afk, but now that I'm addressing your requests, I remembered that I also made a getParentid method for internal use of the mixin but also external. So we can call that method and pass whatever id it returns to us to the getImageUrl method without the need of further options.